WardrobeCloset/App_Start/BundleConfig.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Web;
+using System.Web.Optimization;
+
+namespace WardrobeCloset
+{
+    public class BundleConfig
```

Comment:
Always good to keep configuration separate from your actual runtime code (what we call the "business logic"). Nice!